boards/mulle/include/periph_conf.h

```diff
@@ -265,9 +265,9 @@ static const spi_conf_t spi_config[] = {
         .pin_clk  = GPIO_PIN(PORT_D, 1),
         .pin_cs   = {
             GPIO_PIN(PORT_D, 0),
-            GPIO_UNDEF,
-            GPIO_UNDEF,
-            GPIO_UNDEF,
+            GPIO_PIN(PORT_D, 4),
+            GPIO_PIN(PORT_D, 5),
```

Comment:
Without going into details, are you sure this is `D5` and not `D0`? Unfortunately I don't have a mulle to test.